### PR TITLE
Add a SearchTips class to return a random search tip for the Articles…

### LINKED ISSUE #1674: Article home page -- search tips
See #1599.

We also need a list of search tips so that we can randomly rotate them

### PR DESCRIPTION
**Pull request by jkeck**
… home page.

Closes #1674 

## Random tip
tip1

## Random tip
tip2

## Random tip
tip3

### CODE REVIEW

File: app/assets/stylesheets/modules/home-page.scss [hunk 1]
```diff
@@ -21,6 +21,11 @@
     .panel-heading {
       color: $cardinal-red;
     }
+
+    dt {
+      width: 100%;
+      text-transform: none !important; // Fighting !important in base.scss
```
Comment (jkeck) [inline app/assets/stylesheets/modules/home-page.scss]: 😞

File: config/settings.yml [hunk 2]
```diff
@@ -47,3 +47,26 @@ STANFORD_NETWORK:
   ranges: []
 BITLY:
   ACCESS_TOKEN: 'the_bitly_apikey'
+SEARCH_TIPS:
```
Comment (jkeck) [inline config/settings.yml]: Not super pleased w/ this, but there is a goal to make this somewhat easily editable (i.e. not buried in a class/view).

Happy to hear other possible approaches.

File: app/views/articles/_home_page.html.erb [hunk 3]
```diff
@@ -56,8 +56,7 @@
       <div class="panel panel-default">
         <div class="panel-body">
           <h2>Search tip</h2>
-          <h4><%= t("searchworks.articles.search_tips.heading") %></h4>
-          <p><%= t("searchworks.articles.search_tips.body") %></p>
+          <%= render SearchTips.random %>
```
Comment (jkeck) [inline app/views/articles/_home_page.html.erb]: So this is pretty static and straight forward right now.

Another thought was to render all the tips into the page and use javascript to randomize.  That would have the benefit of allowing some sort of rotation/carousel of tips or a control to update with a new random tip (although that's not necessarily in the design so may be overkill at this time).

File: config/settings.yml (hunk 2 above)
Comment (mejackreed) [reply config/settings.yml]: What about putting in a separate config file and adding source at runtime (initializer?)

https://github.com/railsconfig/config#adding-sources-at-runtime

```ruby
Settings.add_source!("/config/catfacts.yml")
```

**Comment by mejackreed** [comment]
Looks good.. one comment on a potential way to integrate the tips / yml

File: config/settings.yml (hunk 2 above)
Comment (jkeck) [reply config/settings.yml]: Neat!  Didn't know about `add_source!`.  I'll give that a go.

**Comment by jkeck** [comment]
👍 Thanks for the review @mejackreed.  I'll def. have to use that in the future to help organize my settings files.

Let me know if you see anything else!

**Pull request merged**